lib/puck/jar.rb

```diff
@@ -49,7 +49,9 @@ class Jar
     #
     # @param [Hash] configuration
     # @option configuration [String] :extra_files a list of files to include in
-    #   the Jar. The paths must be below the `:app_dir`.
+    #   the Jar. The option can be either an Array, in which case paths must be
+    #   below the `:app_dir`, or a Hash, in which case the file specified by the
+    #   key is included at the path specified by the corresponding value.
```

Comment:
The type of the parameter is wrong – but it looks like it was wrong before too. Perhaps you could change it to `[Array<String>, Hash<String,String>]` (or whatever the right syntax for Yard is)?